tests/assets/instrumented-manual.html

```diff
@@ -0,0 +1,36 @@
+<!DOCTYPE html>
+<!--
+  Copyright 2020 New Relic Corporation.
+  PDX-License-Identifier: Apache-2.0
+-->
+<html>
+  <head>
+    <title>RUM Unit Test</title>
+    {init} {config} 
+    <script>
+      NREUM.init.auto = {
+              ajax: false,
+              jserrors: false,
+              metrics: false,
+              page_action: false,
+              page_view_event: false,
+              page_view_timing: false,
+              session_trace: false,
+              spa: false
+            }
```

Comment:
This is fine, I just needed a place to add this comment 😉 . The file `tools/wdio/plugins/testing-server/default-asset-query.mjs` needs to be updated with the defaults please.